Mathlib/MeasureTheory/Function/LpSeminorm.lean

```diff
@@ -667,12 +685,16 @@ theorem Memℒp.right_of_add_measure {f : α → E} (h : Memℒp f p (μ + ν))
 #align measure_theory.mem_ℒp.right_of_add_measure MeasureTheory.Memℒp.right_of_add_measure
 
 theorem Memℒp.norm {f : α → E} (h : Memℒp f p μ) : Memℒp (fun x => ‖f x‖) p μ :=
-  h.of_le h.AEStronglyMeasurable.norm (eventually_of_forall fun x => by simp)
+  h.of_le h.aestronglyMeasurable.norm (eventually_of_forall fun x => by simp)
 #align measure_theory.mem_ℒp.norm MeasureTheory.Memℒp.norm
 
 theorem memℒp_norm_iff {f : α → E} (hf : AEStronglyMeasurable f μ) :
     Memℒp (fun x => ‖f x‖) p μ ↔ Memℒp f p μ :=
-  ⟨fun h => ⟨hf, by rw [← snorm_norm]; exact h.2⟩, fun h => h.norm⟩
+  ⟨fun h =>
+    ⟨hf, by
+      rw [← snorm_norm]
+      exact h.2⟩,
+    fun h => h.norm⟩
```

Comment:
Any reason you changed this?

Reply:
I haven't changed this. Probably, this is because Mathport was upgraded and output one-line.